frontend/src/views/CampCreate.vue

```diff
@@ -1,53 +1,200 @@
+                            v-for="category in prototypePreview.categories().items"
+                            :key="category._meta.self"
+                          >
+                            <v-list-item-title class="d-flex gap-2 align-baseline">
+                              <CategoryChip
+                                :category="category"
+                                class="mx-1 flex-shrink-0"
+                                dense
+                              />
+                              <span class="font-weight-medium">{{ category.name }}</span>
+                              <small class="blue-grey--text">{{
+                                category
+                                  .preferredContentTypes()
+                                  .items.map((item) =>
+                                    $tc('contentNode.' + camelCase(item.name) + '.name')
+                                  )
+                                  .join(', ') || $tc('views.campCreate.noContent')
+                              }}</small>
+                            </v-list-item-title>
+                          </v-list-item>
+                        </v-list>
+                      </v-expansion-panel-content>
+                    </v-expansion-panel>
+                    <v-expansion-panel>
+                      <v-expansion-panel-header>
+                        <h3>Activity states preview</h3>
+                      </v-expansion-panel-header>
+                      <v-expansion-panel-content>
+                        <v-list class="py-0 mx-n4">
+                          <v-list-item
+                            v-for="(
+                              progressLabel, idx
+                            ) in prototypePreview.progressLabels().items"
+                            :key="progressLabel._meta.self"
+                          >
+                            <v-list-item-title class="d-flex gap-2 align-baseline">
+                              <v-avatar color="grey lighten-2 subtitle-2" size="24"
+                                >{{ idx + 1 }}
+                              </v-avatar>
+                              {{ progressLabel.title }}
+                            </v-list-item-title>
+                          </v-list-item>
+                        </v-list>
+                      </v-expansion-panel-content>
+                    </v-expansion-panel>
+                  </v-expansion-panels>
+                </v-sheet>
+                <v-divider v-else />
+                <ContentActions>
+                  <v-btn text color="secondary" :disabled="isSaving" @click="step--">
+                    <v-icon left>mdi-arrow-left</v-icon>
+                    {{ $tc('global.button.back') }}
+                  </v-btn>
+                  <v-spacer />
+                  <ButtonCancel :disabled="isSaving" @click="$router.go(-1)" />
+                  <ButtonAdd v-if="valid" type="submit" :loading="isSaving">
```

Comment:
On mobile, this button flows onto the next row.
Is this on purpose?

![grafik](https://github.com/ecamp/ecamp3/assets/1506818/7a7a94e5-60ae-4f67-ad3b-5844e02bc5c1)